src/demos/datepicker/fuzzy-datepicker-demo.component.html

```diff
@@ -0,0 +1,133 @@
+<h2
+  class="sky-section-heading"
+>
+  Standard fuzzy datepicker
+</h2>
+
+<div
+  style="max-width: 50%"
+>
+  <sky-datepicker>
+    <input
+      skyFuzzyDatepickerInput
+      [futureDisabled]="futureDisabled"
+      [dateFormat]="dateFormat"
+      [disabled]="disabled"
+      [maxDate]="maxDate"
+      [minDate]="minDate"
+      [skyFuzzyDatepickerInput]
+      [skyDatepickerNoValidate]="noValidate"
+      [startingDay]="startingDay"
+      [yearRequired]="yearRequired"
+      [(ngModel)]="selectedDate"
+      #date="ngModel"
+    />
+  </sky-datepicker>
+
+  <p *ngIf="date.errors && date.errors.skyFuzzyDate && (date.dirty || date.touched)"
+    class="sky-error-label"
+  >
+    <span *ngIf="date.errors.skyFuzzyDate?.invalid">
+      Please enter a valid date.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.yearRequired">
+      Please select a year.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.futureDisabled">
+      Date cannot be in the future.
+    </span>
+    <span *ngIf="date.errors.skyFuzzyDate?.maxDate">
+      The current date is beyond the maximum allowed date.
```

Comment:
I wrote around "minimum" and "maximum" in the property description because that's not really how people talk about dates. I think it's fine to use "min" and "max" in the property names, but I'd suggest rewording to avoid the terminology in our error examples. Also, these error message strike me as too vague. We tell users that dates are too early or too late, but we don't tell them what the thresholds are, so we leave them to guess what the acceptable dates are. The error message shouldn't leave them guessing. What do you think about changing this error message to `This date is earlier than allowed. Please select a date after [min date].`?

Reply:
Done.

Reply:
See question that follows on date format.

Reply:
Date formatted.